app/src/hydrate.ts

```diff
@@ -50,6 +51,7 @@ export async function hydrate(): Promise<void> {
 	const appStore = useAppStore();
 	const userStore = useUserStore();
 	const permissionsStore = usePermissionsStore();
+	const { refresh: hydrateTranslationStrings } = useTranslationStrings();
```

Comment:
Maybe we can just rename the exported value from `useTranslationStrings`. Makes it easier to find and replace later if needed

Reply:
Nah, forget it. It's cleaner to leave it as refresh in the context of that composable 👍🏻